pkg/loadbalancer/experimental/bpf_reconciler.go

```diff
@@ -18,9 +18,11 @@ import (
 	"golang.org/x/sys/unix"
```

Comment:
> > Conformance Kind Envoy Embedded / Installation and Connectivity Test (pull_request) Failing after 16m
> 
> ```
> 📋 Test Report [cilium-test-5]
> ❌ 1/13 tests failed (1/196 actions), 6 tests skipped, 0 scenarios skipped:
> Test [client-egress-l7-set-header-port-range]:
>   ❌ client-egress-l7-set-header-port-range/pod-to-pod-with-endpoints/curl-ipv4-2-auth-header-required: cilium-test-5/client3-795488bf5-8lcsb (10.244.0.214) -> curl-ipv4-2-auth-header-required (10.244.1.250:8080)
> ```
> 
> I see very similar test failures for an empty PR https://github.com/cilium/cilium/actions/runs/12429666814/job/34703608166?pr=36744, so it's most likely some flakiness or regression on `main`.

Yeah, it succeeded now (which is probably not related to changes introduced in the meantime), although 5 other sub-tests failed

Status | Package | Time (seconds)
-- | -- | --
❌ | client-egress-l7-tls-headers-sni | 194.916
❌ | client-egress-tls-sni | 441.098
❌ | no-policies | 635.625
❌ | seq-client-egress-l7-tls-headers | 195.541
❌ | seq-client-egress-l7-tls-headers-port-range | 195.290

(which I suppose succeeded before).